service/src/main/java/org/apache/griffin/core/metric/MetricStoreImpl.java

```diff
@@ -56,7 +56,8 @@ Licensed to the Apache Software Foundation (ASF) under one
 @Component
 public class MetricStoreImpl implements MetricStore {
 
-    private static final String INDEX = "griffin";
+    @Value("${index}")
+    private String INDEX;
```

Comment:
Java naming convention for fields and variables is [lowercase](https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html)